README.md

```diff
@@ -360,6 +360,13 @@ transloco-keys-manager find --add-missing-keys
 transloco-keys-manager find -a
 ```
 
+- `emitErrorOnExtraKeys`: Return exit code if extra keys were found by the detective (defaults to `false`)
```

Comment:
Emit an error and exit the process if extra keys were found (defaults to `false`)

Reply:
Done